pages.php

```diff
@@ -32,6 +32,32 @@
     <head>
     </head>
     <body>
-        <?php /* Use PHP's include() method to render the correct page content in here */ ?>
+        <?php /* Use PHP's include() method to render the correct page content in here */ 
+            echo'
+                <a href="/pages.php?page=home"> ++Home++ </a>
+                <a href="/pages.php?page=about"> ++About++ </a>
+                <a href="/pages.php?page=interests"> ++Interests++ </a>';
```

Comment:
@kko31623 you don't need to use `echo` to put these links in the page. They can go before the `<?php` tag and then they're just part of the normal HTML.